tests/unit/components/pacbio/PacbioWell.spec.js

```diff
@@ -15,6 +15,7 @@ describe('Well.vue', () => {
     storeWell.insert_size = 123
     storeWell.on_plate_loading_concentration = 234
     storeWell.sequencing_mode = "CCS"
+    storeWell.pre_extension_time = "2"
```

Comment:
I notice you've got some integers on the other attributes - pre_extension_time should really be an integer too (it is in the database)